setup.cfg

```
@@ -114,7 +114,7 @@ all =
     napari-plugin-manager >=0.1.0a1, <0.2.0
 # optional (i.e. opt-in) packages, see https://github.com/napari/napari/pull/3867#discussion_r864354854
 optional =
-    triangle
+    triangle ; platform_machine != 'arm64'
```

```suggestion
    triangle ; platform_machine != 'arm64' # no wheels
```
Does a comment like this work?

---

added